index.html

```diff
@@ -0,0 +1,138 @@
+                </li>
+                <li>
+                    <a class="nav-item" href="#" tabindex="5">Join</a>
+                </li>
+                <li>
+                    <a class="nav-item" href="#" tabindex="6">Contact</a>
+                </li>
+            </ul>
+        </div>
+        </nav>
+
+    <div id="wrapper">
+
+        <section class="intro" id="intro" tabindex="7">
+            <div class="intro-text">
+
+                <h1 aria-label="Annual International Science Research Conference 2023" class="text-bg fw-bold p-1">
+                    Annual International
+                    <br />
+                    Science Research Conference 2023
+                </h1>
+                <h2 aria-label="Hello! Sharing world">"Hello! Sharing world"</h2>
+                <p aria-label="I can help you build a product , feature or website Look through some of
+            my work and experience! If you like what you see and have a project you
+            need coded, don’t hestiate to contact me.">
+                    A joyful celebreation believing in the value of openness and sharing creating a
+                    positive change with people from over 80 countries is taking place in February, in Australia.
+                </p>
+                <h3>
+                    2023.02.29-31
+                </h3>
+                <p id="address">
+                    @ Melboure Information University, Mcdonalds Building
+                </p>
+            </div>
+
+        </section>
+
+
+        <section class="about" id="about">
+            <h1>Main Programs</h1>
+            <hr />
+                    <div id="programs" aria-label="Lecture">
+                        <div class="program">
+                            <img src="./img/lecture.png" alt="lecture Logo" />
+                            <h2>Lecture</h2>
+                            <p>Listen to the speakers from various countries
+                                about the messages of sharing and opening.
+                            </p>
+                        </div>
+                        <div class="program">
+                            <img src="./img/route.png" alt="route Logo" />
+                            <h2>Route</h2>
+                            <p>Easy to find in downtown Melboure with
+                                lots of attractions to visit during your 
+                                spare time.
+                            </p>
+                        </div>
+                        <div class="program">
+                            <img src="./img/hotel.png" alt="hotel Logo" />
+                            <h2>Hospitality</h2>
+                            <p>Lots of choices of outstanding hotels 
+                                 to satisfy your needs during Conference.
+                            </p>
+                        </div>
+                        <div class="program">
```

Comment:
- [ ] The mobile view of this section's individual articles should have their elements organized in a horizontal row, with the image at the left, the title in the center, and the description at the right just as the attached Behance image.
## Behance image
![image](https://user-images.githubusercontent.com/86200356/232958415-c0d55168-afd0-48ff-82e1-7107a33aa431.png)
## Your design
![image](https://user-images.githubusercontent.com/86200356/232958709-772ed78b-2826-4005-9a0f-439078f473d3.png)